newIDE/app/src/ObjectEditor/Editors/Model3DEditor.js

```diff
@@ -0,0 +1,269 @@
+      initialResourceName={property.getValue()}
+      onChange={value => onChangeProperty(propertyName, value)}
+      resourcesLoader={resourcesLoader}
+      fullWidth
+    />
+  );
+};
+
+const Model3DEditor = ({
+  objectConfiguration,
+  project,
+  layout,
+  resourceManagementProps,
+}: EditorProps) => {
+  const forceUpdate = useForceUpdate();
+  const properties = objectConfiguration.getProperties();
+
+  const onChangeProperty = React.useCallback(
+    (property: string, value: string) => {
+      objectConfiguration.updateProperty(property, value);
+      forceUpdate();
+    },
+    [objectConfiguration, forceUpdate]
+  );
+
+  return (
+    <ColumnStackLayout noMargin>
+      <PropertyResourceSelector
+        objectConfiguration={objectConfiguration}
+        propertyName="modelResourceName"
+        project={project}
+        resourceManagementProps={resourceManagementProps}
+      />
+      <Text size="block-title" noMargin>
+        <Trans>Default orientation</Trans>
+      </Text>
+      <ResponsiveLineStackLayout expand noColumnMargin>
+        {['rotationX', 'rotationY', 'rotationZ'].map(propertyName => (
+          <PropertyField
+            objectConfiguration={objectConfiguration}
+            propertyName={propertyName}
+          />
+        ))}
+      </ResponsiveLineStackLayout>
+      <Text size="block-title" noMargin>
+        <Trans>Default size</Trans>
+      </Text>
+      <ResponsiveLineStackLayout expand noColumnMargin>
+        {['width', 'height', 'depth'].map(propertyName => (
+          <PropertyField
+            objectConfiguration={objectConfiguration}
+            propertyName={propertyName}
+          />
+        ))}
+      </ResponsiveLineStackLayout>
+      <ColumnStackLayout noMargin expand>
+        <PropertyCheckbox
+          objectConfiguration={objectConfiguration}
+          propertyName="keepAspectRatio"
+        />
+        <SelectField
+          value={properties.get('materialType').getValue()}
+          floatingLabelText={properties.get('materialType').getLabel()}
+          helperMarkdownText={properties.get('materialType').getDescription()}
+          onChange={(event, index, newValue) => {
+            onChangeProperty('materialType', newValue);
+          }}
+        >
+          <SelectOption
+            label={t`Always lighted (no light needed)`}
+            value="AlwaysLighted"
+            key="AlwaysLighted"
+          />
+          <SelectOption
+            label={t`Emit all ambient light`}
+            value="EmitAllAmbientLight"
+            key="EmitAllAmbientLight"
+          />
+          <SelectOption label={t`No change`} value="NoChange" key="NoChange" />
+        </SelectField>
+        {properties.get('materialType').getValue() !== 'AlwaysLighted' &&
+          !hasLight(layout) && (
+            <AlertMessage kind="error">
+              <Trans>
+                For standard materials reacting to lights, make sure to set up a
```

Comment:
We need to use the same terminology as in the dropdown. So if we use "standard materials" here, we should use standard material.
With my suggestion, we can use "For realistic material (also called standard material), reacting to lights, make sure [...]"